src/visualBuilder/eventManager/useCollab.ts

```diff
@@ -45,19 +60,32 @@ export const useCollab = () => {
         }
     );
 
-    visualBuilderPostMessage?.on(
+    const collabThreadRemove = visualBuilderPostMessage?.on(
         VisualBuilderPostMessageEvents.COLLAB_THREAD_REMOVE,
         (data: any) => {
             const { threadUid } = data.data;
             removeCollabIcon(threadUid);
         }
     );
 
-    visualBuilderPostMessage?.on(
+    const collabThreadReopen = visualBuilderPostMessage?.on(
         VisualBuilderPostMessageEvents.COLLAB_THREAD_REOPEN,
         (data: any) => {
             const thread = data.data.thread;
-            generateThreadFromData(thread);
+            const result = generateThread(thread, { isNewThread: false });
+            if (result !== undefined) {
```

Comment:
can be simplified to
```suggestion
            if (result) {
```

Reply:
sure